pkg/dotnet/core_deps/parse.go

```diff
@@ -31,30 +50,60 @@ func (p *Parser) Parse(r dio.ReadSeekerAt) ([]types.Library, []types.Dependency,
 	}
 
 	var libraries []types.Library
-	for nameVer, lib := range depsFile.Libraries {
-		if !strings.EqualFold(lib.Type, "package") {
+	var deps []types.Dependency
+	for pkgNameVersion, target := range depsFile.Targets[depsFile.RuntimeTarget.Name] {
```

Comment:
I don't have a lot of exp with .Net, but found next case:
`RuntimeTarget.Name` from depsFile != `RuntimeTarget.Name` from `targets`:
https://github.com/dotnet/cli/blob/f0ee1e44114f161ce9268103d18ce32d2a995d0f/Documentation/specs/runtime-configuration-file.md?plain=1#L48
https://github.com/dotnet/cli/blob/f0ee1e44114f161ce9268103d18ce32d2a995d0f/Documentation/specs/runtime-configuration-file.md?plain=1#L61

Is this possible? or is this a mistake?

Reply:
I believe this is a mistake. I've never been able to find another example of this occurring in a real .json file (not an example in a `.md`)

Reply:
Okey. Thanks!